### PR TITLE
7.2.0

### PR DESCRIPTION
**Pull request by robertohuertasm**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+  - Fix: `Patch` icon has been improved. ([@JimiC](https://github.com/JimiC) in [#703](https://github.com/vscode-icons/vscode-icons/pull/703))
+  - Fix: `Slim` icon has been improved. ([@JimiC](https://github.com/JimiC) in [#701](https://github.com/vscode-icons/vscode-icons/pull/701))
+  - Enhancement: New logo for the extension (svg). ([@JimiC](https://github.com/JimiC) in [#689](https://github.com/vscode-icons/vscode-icons/pull/689))
+  - Enhancement: New batch of svg icons. ([@JimiC](https://github.com/JimiC) in [#687](https://github.com/vscode-icons/vscode-icons/pull/687))
+  - Enhancement: Angular autodetection cancellation improved.  ([@JimiC](https://github.com/JimiC) in [#679](https://github.com/vscode-icons/vscode-icons/pull/679))
+  - Fix: Disabled folders are never shown.  ([@robertohuertasm](https://github.com/robertohuertasm) in [#678](https://github.com/vscode-icons/vscode-icons/pull/678))
+  - Enhancement: `package.json`update to use vscode engine <=1.5.0.  ([@JimiC](https://github.com/JimiC) in [#675](https://github.com/vscode-icons/vscode-icons/pull/675))
```
Comment (JimiC) [inline CHANGELOG.md]: Correction: it's >=1.5.0

Comment (JimiC) [inline CHANGELOG.md]: And put some spacing between `package.json` and `update`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # Change Log
 
 ## 7.2.0 (February 4, 2017)
+  - Enhancement: `textmate`support. ([@JimiC](https://github.com/JimiC) in [#723](https://github.com/vscode-icons/vscode-icons/pull/723))
+  - Fix: `.editorconfig` instructed not to modify images. ([@JimiC](https://github.com/JimiC) in [#722](https://github.com/vscode-icons/vscode-icons/pull/722))
+  - Fix: exclude npm logs, vim support, fix example generator and supportedExtensions file optimization. ([@JimiC](https://github.com/JimiC) in [#721](https://github.com/vscode-icons/vscode-icons/pull/721))
```
Comment (JimiC) [inline CHANGELOG.md]: Capital wording in beginning.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,10 @@
 # Change Log
 
 ## 7.2.0 (February 4, 2017)
+  - Enhancement: `textmate`support. ([@JimiC](https://github.com/JimiC) in [#723](https://github.com/vscode-icons/vscode-icons/pull/723))
```
Comment (JimiC) [inline CHANGELOG.md]: Include spacing.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,26 @@
 # Change Log
 
+## 7.2.0 (February 4, 2017)
```
Comment (JimiC) [inline CHANGELOG.md]: It's Feb 5 now.

File: CHANGELOG.md (hunk 2 above)
Comment (JimiC) [inline CHANGELOG.md]: line 4 still pending fix.

File: CHANGELOG.md (hunk 3 above)
Comment (JimiC) [inline CHANGELOG.md]: line 4 still pending fix.

**Comment by JimiC** [comment]
The writing to the workspace settings will be fixed with this release as you know.

**Comment by JimiC** [comment]
If project detection creates to much fuzz and support for you just remove it.

**Comment by robertohuertasm** [comment]
I totally missed it. I don't know what's happening with vscode but now when I use the open accent it doesn't put the space automatically as before.

**Comment by JimiC** [comment]
I hoped you could provide the Angular `controller` icon before this release.

**Comment by robertohuertasm** [comment]
About the project detection, I know @JimiC, and I'm not thinking about removing it for the moment. I think it's a good utility but maybe it shouldn't be enabled by default (just thinking about alternatives). I'd like to find the time to discuss it with you in Slack. Maybe during the week if that's fine for you.

**Comment by JimiC** [comment]
From my perspective, Angular icons shouldn't be enabled by default, simply because the majority of the projects aren't Angular based.

**Comment by robertohuertasm** [comment]
Yes, maybe you're right. Definetely, you have a strong point. I've created the `Angular controller` icons but I think that maybe we should deliver them once we have `Angular 1` detection solved. What do you think?

**Comment by robertohuertasm** [comment]
I've finally published the `controller` icons in `7.2.0`. 

Just as a side note, the guy of the awful review has reconsidered and has given us 5 stars again! 😁

**Pull request merged**